ecosystem/platform/server/config/environments/development.rb

```diff
@@ -75,7 +75,7 @@
 
   # Suppress logger output for asset requests.
   config.assets.quiet = true
-
+  config.assets.debug = true
```

Comment:
is this intentional?

Reply:
yeah! that debug flag is on purpose: `/config/environments/development.rb` rails has different configs per env; the assets debug flag was the missing piece of why I had to kill my server every time I needed to recompile tailwind :-)